src/component/plotly-dash-preview/parse.js

```diff
@@ -0,0 +1,39 @@
+const isNonEmptyString = require('../../util/is-non-empty-string')
+
+/**
+ * @param {object} body : JSON-parsed request body
+ *  - url
+ *  - share_key
+ *  - parameters
+ * @param {object} opts : component options
+ * @param {function} sendToRenderer
+ * - errorCode
+ * - result
+ */
+function parse (body, opts, sendToRenderer) {
+  const result = {}
+
+  const errorOut = (code, msg) => {
+    result.msg = msg
+    sendToRenderer(code, result)
+  }
+
+  if (isNonEmptyString(body.url)) {
+    result.url = body.url
+  } else {
+    return errorOut(400, 'invalid url')
+  }
+
+  if (isNonEmptyString(body.format) && (body.format === 'png' || body.format === 'pdf')) {
+    result.format = body.format
+  } else {
+    // Default
+    result.format = 'png'
+  }
```

Comment:
We don't need to support other image types, since these are only going to be used for PDF exports and Organize thumbnails ..